docs/source/conf.py

```diff
@@ -107,7 +107,7 @@ def find_version(*file_paths):
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = "en"
```

Comment:
`language = None` is no longer valid in Sphinx 5, either set it to `"en"` or remove the setting.